composer.json

```diff
@@ -14,7 +14,7 @@
     },
     "minimum-stability": "dev",
     "require": {
-        "php": ">=5.6.0",
+        "php": ">=7.2",
```

Comment:
What's the reason to raise this requirement?

Reply:
Because in the message result:

doctrine/lexer 1.1.0 requires php ^7.2 -> your PHP version (7.1.33) does not satisfy that requirement.


doctrine/lexer require php 7.2

Reply:
Interesting. What is requiring doctrine/lexer?

Reply:
Is this the same as https://github.com/yiisoft/yii2-app-basic/issues/237#issuecomment-716359928 ?